### DISCUSSION TITLE
限定公開→全体公開と切り替えたタイミングで最新記事としてRSSが更新されるようにしてほしい

### DISCUSSION THREAD

**Discussion by noriyuki-amari-he** [increments/qiita-discussions#235] [要望・アイデア・不具合]
お世話になっています。

会社の技術ブログとして現在以下のフローで利用させていただいています。

1. 限定公開の記事を作成
2. 社内レビュー
3. 記事の公開設定を全体に変更
4. 組織に紐付け

上記のように記事を公開した際に組織（organization）RSSのpublishedの時刻が限定公開の時刻で載ってしまい、RSSリーダー（Slack）が最新のものとして認識できなくなってしまいます。

ここは、RSSに追加されるタイミングがpublishedになるべきかと思います。
つまり

1. 限定公開の段階でRSSに乗る
2. 全体公開の段階での時刻をRSSに載せる

のいずれかが自然だと思いました。

上記ご検討いただけますと幸いです。

**Comment by noriyuki-amari-he** [comment]
上記問題ありませんでした。そのままクローズいただければと思います。

**Reply by tanaka-asami415** [reply]
@noriyuki-amari-he 
ありがとうございます🙏
問題ないとのことでしたが、確認して改めてこちらでご連絡させていただきます🙇

**Reply by kyntk** [reply]
@noriyuki-amari-he
調査をしたところ、各entryのpublishedが限定共有記事の作成日時になっていました。
並び順は公開日時になっていますが、entryのpublishedが正しくないので修正します！

**Reply by kyntk** [reply]
@noriyuki-amari-he 
publishedの値の修正も完了いたしました！
https://qiita.com/release-notes#2022-06-07

こちらのDiscussionはクローズさせていただきます 🙏
